spec/compiler/crystal/tools/context_spec.cr

```diff
@@ -1,26 +1,24 @@
 require "spec"
 require "yaml"
-require "../../../../src/compiler/crystal/**"
```

Comment:
@bcardiff @RX14 I'll fix to use `require "../../../spec_helper"` instead of `require "compiler/crystal/**"` because other compiler's spec uses it. (And `spec/spec_helper.cr` requires compiler files with relative path. If you want to fix it, please open another pull request or issue.)